docs/cdp/cdp.md

```diff
@@ -0,0 +1,20 @@
+# Customer Data Platform (CDP) [[% include 'snippets/experience_badge.md' %]] [[% include 'snippets/commerce_badge.md' %]]
```

Comment:
Not sure if I understand the badge correctly - if it means CDP is available from Experience edition then this is wrong as it's a standalone package that can be installed alongside every product edition (Content, Experience, Commerce). Maybe we should add Content badge as well.

Reply:
So let's remove them.